src/parameters.toml

```diff
@@ -2214,6 +2214,24 @@ value = 355
 type = "float"
 description = "Maximum temperature in the lookup table for optical properties in RRTMGP"
 
+# RCEMIP
```

Comment:
Thanks @cmschmitt519. Could you also please bump the patch version in `Project.toml` and update the `NEWS.md` file highlighting the additional parameters that you're introducing?